class-plugin.php

```diff
@@ -35,15 +43,47 @@ public function action_admin_init() {
 
 	}
 
+	/**
+	 * Display Network Settings for Google Tag Manager
+	 */
+	public function show_network_settings() { ?>
+		<h3><?php esc_html_e( 'Network Google Tag Manager', 'hm_gtm' ); ?></h3>
+		<table id="menu" class="form-table">
+			<tr valign="top">
+				<th scope="row"><?php esc_html_e( 'Container ID', 'hm_gtm' ); ?></th>
+				<td>
+					<label for="hm_network_gtm_id">
```

Comment:
This `<label>` doesn't make sense here as it just contains the input - it should go around the text in the `<th>` tag above.